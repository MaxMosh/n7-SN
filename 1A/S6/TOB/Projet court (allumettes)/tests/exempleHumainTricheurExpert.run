java allumettes.Jouer -confiant Xavier@humain Ordinateur@expert <<EOF
1
3
triche
3
EOF
