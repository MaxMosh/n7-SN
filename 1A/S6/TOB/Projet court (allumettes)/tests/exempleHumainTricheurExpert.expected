Allumettes restantes : 13
Xavier, combien d'allumettes ? Xavier prend 1 allumette.

Allumettes restantes : 12
Ordinateur prend 3 allumettes.

Allumettes restantes : 9
Xavier, combien d'allumettes ? Xavier prend 3 allumettes.

Allumettes restantes : 6
Ordinateur prend 1 allumette.

Allumettes restantes : 5
Xavier, combien d'allumettes ? [Une allumette en moins, plus que 4. Chut !]
Xavier, combien d'allumettes ? Xavier prend 3 allumettes.

Allumettes restantes : 1
Ordinateur prend 1 allumette.

Ordinateur perd !
Xavier gagne !
